eth/eventhandler/handlers.go

```diff
@@ -469,6 +468,48 @@ func (eh *EventHandler) handleFeeRecipientAddressUpdated(txn basedb.Txn, event *
 	return r != nil, nil
 }
 
+type ExitDescriptor struct {
+	PubKey         phase0.BLSPubKey
+	ValidatorIndex phase0.ValidatorIndex
+	Slot           phase0.Slot
+}
+
+func (eh *EventHandler) handleValidatorExited(txn basedb.Txn, event *contract.ContractValidatorExited) (*ExitDescriptor, error) {
```

Comment:
missing in this function: does the address who triggered this tx really the address that registered this validator?

Reply:
Well spotted, thanks. Added